src/Simulations/Simulations.jl

```diff
@@ -25,4 +25,9 @@ include("simulation.jl")
 include("run.jl")
 include("time_step_wizard.jl")
 
+# Implementation of the diagnostic for computing the dissipation rate
+include("VarianceDissipationComputations/VarianceDissipationComputations.jl")
```

Comment:
I think this belongs in Models/

Reply:
I can move it in models

Reply:
Remember that the Simulations module is supposed to be designed so that it can be lifted out.

One change we can make if we want to clean things up is to put the Simulations before Models, if you want to extend Callback in Models.